bin/code_picardQC.sh

```
@@ -0,0 +1,189 @@
+# if run_CollectTargetedPcrMetrics is true
+if [[ "$run_CollectTargetedPcrMetrics" == true ]]; then
+# Call Picard CollectTargetedPcrMetrics
+collect_targeted_pcr_metrics
+fi
+
+# if CollectRnaSeqMetrics is true
+if [[ "$run_CollectRnaSeqMetrics" == true ]]; then
+# use ref flat file if provided else convert gtf to refl flat from gtf
+if [ -z "$ref_annot_refflat" ]; then # when there's NO refflat
+	echo "create refflat file from gtf file in CTAT bundle"
+	lib_dir=$(echo $fasta_index_name | cut -d "." -f 1,2)
```

Shellcheck (0.10.0)

[warning] fasta_index_name is referenced but not assigned.

(SC2154)